awx/api/generics.py

```diff
@@ -197,6 +200,8 @@ def finalize_response(self, request, response, *args, **kwargs):
                 logger.warning(status_msg)
         response = super(APIView, self).finalize_response(request, response, *args, **kwargs)
         time_started = getattr(self, 'time_started', None)
+        response['X-API-AWX-Version'] = get_awx_version()
```

Comment:
bikeshedding... but could we not *call* it AWX in the header key? Maybe `X-API-Product-Version` ?

Reply:
Yea, agreed.